### PR TITLE
Update Spaces menu ordering and capitalisation

### LINKED ISSUE #5524: Update Spaces menu ordering and capitalisation
The space menu in left panel -> ⋮needs to be reordered to be:

- Invite people
- Explore rooms
- Members
- Settings
- Add rooms
- Add space
- Leave

Note change in capitalisation for "space".

### PR DESCRIPTION
**Pull request by fedrunov**
## Type of change

- [ ] Feature
- [ ] Bugfix
- [ ] Technical
- [x] Other : misc

## Content

Update Spaces menu ordering and capitalisation

## Motivation and context

fixes #5524 

## Screenshots / GIFs

image

## Tests

- open left slider menu
- open space menu

## Tested devices

- [ ] Physical
- [x] Emulator 
- OS version(s): 31

## Checklist

- [ ] Changes has been tested on an Android device or Android emulator with API 21
- [ ] UI change has been tested on both light and dark themes
- [x] Accessibility has been taken into account. See https://github.com/vector-im/element-android/blob/develop/CONTRIBUTING.md#accessibility
- [x] Pull request is based on the develop branch
- [x] Pull request includes a new file under ./changelog.d. See https://github.com/vector-im/element-android/blob/develop/CONTRIBUTING.md#changelog
- [x] Pull request includes screenshots or videos if containing UI changes
- [ ] Pull request includes a [sign off](https://matrix-org.github.io/synapse/latest/development/contributing_guide.html#sign-off)
- [x] You've made a self review of your PR
- [ ] If you have modified the screen flow, or added new screens to the application, you have updated the test [UiAllScreensSanityTest.allScreensTest()](https://github.com/vector-im/element-android/blob/main/vector/src/androidTest/java/im/vector/app/ui/UiAllScreensSanityTest.kt#L73)

### CODE REVIEW

**Comment by fedrunov** [comment]
We probably also need to change capitalisation in other locales, what is the best way to do so? Just update manually in string resources and commit to this PR or somehow else?

**Review by ouchadam — APPROVED** [review]
LGTM! 

I'm assuming the space sanity tests are already using the strings so we don't need to update them?

File: vector/src/main/res/layout/bottom_sheet_space_settings.xml [hunk 1]
```diff
@@ -79,6 +79,17 @@
...
+    <im.vector.app.core.ui.views.BottomSheetActionButton
+        android:id="@+id/exploreRooms"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        app:actionTitle="@string/space_explore_activity_title"
+        app:leftIcon="@drawable/ic_explore"
+        app:tint="?vctr_content_secondary"
+        app:titleTextColor="?vctr_content_primary"
+        tools:actionDescription="" />
+
```
Comment (ouchadam) [inline vector/src/main/res/layout/bottom_sheet_space_settings.xml:91]: looks like an extra bonus line was added

**Comment by fedrunov** [comment]
> I'm assuming the space sanity tests are already using the strings so we don't need to update them?

What do you mean? 🤔

**Comment by ouchadam** [comment]
> > I'm assuming the space sanity tests are already using the strings so we don't need to update them?
> 
> What do you mean? thinking

with order/value change, do we need to update https://github.com/vector-im/element-android/blob/develop/vector/src/androidTest/java/im/vector/app/ui/UiAllScreensSanityTest.kt#L112 ? :thinking:

**Comment by fedrunov** [comment]
> with order/value change, do we need to update https://github.com/vector-im/element-android/blob/develop/vector/src/androidTest/java/im/vector/app/ui/UiAllScreensSanityTest.kt#L112 ? 🤔

got it, no, tests are relying on widget ids, so nothing will change there

**Pull request merged**